examples/ucx/local_cuda_cluster.py

```diff
@@ -35,25 +38,41 @@ def main(
 ):
 
     enable_rdmacm = False
-    ucx_net_devices = None
+    client_devices = worker_devices = None
 
     if enable_infiniband:
         enable_rdmacm = True
-        ucx_net_devices = "auto"
+        client_devices = "mlx5_0:1"
+        worker_devices = "auto"
 
     if (enable_infiniband or enable_nvlink) and not interface:
-        raise ValueError("Interface must be specified if using NVLink or Infiniband")
+        raise ValueError(
+            "Interface must be specified if NVLink or Infiniband are enabled"
+        )
+
+    # set up environment
+    initialize(
+        enable_tcp_over_ucx=True,
+        enable_nvlink=enable_nvlink,
+        enable_infiniband=enable_infiniband,
+        enable_rdmacm=enable_rdmacm,
+        net_devices=client_devices,
+    )
```

Comment:
Sorry if my previous comment was unclear, but this is what I meant, `initialize` isn't necessary here because `LocalCUDACluster` does that internally: https://github.com/rapidsai/dask-cuda/blob/250260e54e0b3c4065bf7bba77cb13f7b032f030/dask_cuda/local_cuda_cluster.py#L257-L264 . Because the `Client` will then be using device 0, leaving `ucx_net_devices="auto"` will also correctly select the client's device.

In short, we only need https://github.com/rapidsai/dask-cuda/pull/551/commits/79b7d5d03574bd68c821cd45edf51e06af9abe52#diff-d9b0b02a0c749f46db506de5db50c2944b96f42895a621ac051539cc5f0da484R3 and https://github.com/rapidsai/dask-cuda/pull/551/commits/79b7d5d03574bd68c821cd45edf51e06af9abe52#diff-d9b0b02a0c749f46db506de5db50c2944b96f42895a621ac051539cc5f0da484R73-R75 from this commit, the rest can be reverted to how it was previously.

Reply:
Ah okay I understand - in that case I should probably alter the [Client](https://github.com/rapidsai/dask-cuda/blob/branch-0.19/docs/source/ucx.rst#client) section of the UCX docs, which also makes this mistake.

Reply:
Oops, that was my fault for not noticing it when you updated the docs, sorry for the extra work. 🙁

Reply:
No worries! Was a quick fix 😃

BTW, do GH review comments generally not give out notifications? I've found myself missing several comments (yours included) because I had to refresh a page to get them to appear.

Reply:
I usually get notifications, but I rely on email notifications. I'm not sure about the actual GH notifications, I stopped using them because they force you to actually load the issue/PR to see what's going on, whereas on email I can just navigate through most of those unimportant ones much more quickly.